src/main/java/org/sagebionetworks/web/client/widget/table/modal/fileview/FileViewOptions.java

```diff
@@ -13,23 +16,33 @@
 	}
 
 	Widget widget;
+	CookieProvider cookieProvider;
+
 	@UiField
 	CheckBox includeFilesCb;
 	@UiField
 	CheckBox includeFoldersCb;
 	@UiField
 	CheckBox includeTablesCb;
+	@UiField
+	CheckBox includeDatasetsCb;
 
 	@Inject
-	public FileViewOptions(Binder binder) {
+	public FileViewOptions(Binder binder, CookieProvider cookies) {
 		widget = binder.createAndBindUi(this);
+		cookieProvider = cookies;
 	}
 
 	@Override
 	public Widget asWidget() {
 		return widget;
 	}
 
+	// TODO: This method can be removed when Datasets is out of experimental mode.
+	public void configure() {
+		includeDatasetsCb.setVisible(DisplayUtils.isInTestWebsite(cookieProvider));
+	}
```

Comment:
Not sure if there's a better way to make this check It's only guaranteed to rerun on page refresh so it may not appear/disappear when initially toggling Experimental Mode. I'm not super worried about it since we're hopefully not super far away from Datasets being released